Hackney.Core/Hackney.Core.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
This project should be a class library, not an exe.

Reply:
This was suggested in the github documentation
[https://docs.github.com/en/packages/guides/configuring-dotnet-cli-for-use-with-github-packages#publishing-a-package-using-a-nugetconfig-file](https://docs.github.com/en/packages/guides/configuring-dotnet-cli-for-use-with-github-packages#publishing-a-package-using-a-nugetconfig-file)

Reply:
Weird...